third_party/protobuf/3.0.0/src/google/protobuf/stubs/atomicops_internals_generic_gcc.h

```diff
@@ -128,6 +128,24 @@ inline Atomic64 NoBarrier_CompareAndSwap(volatile Atomic64* ptr,
   return old_value;
 }
 
+inline Atomic64 NoBarrier_AtomicIncrement(volatile Atomic64* ptr,
```

Comment:
We intend `third_party/protobuf/<version>` to be the exact copy of protobuf's <version> directory (see https://github.com/bazelbuild/bazel/blob/master/third_party/protobuf/README.md). So these files should not be changed as part of ths PR